### PR TITLE
HOFF-409-node: update only node image and drone trivy scan and cron jobs

### PR DESCRIPTION
**Pull request by gadorcruz**
## What?
Update the node image from 18 to 20.16.0 to fix the image vulnerabilities. Also, update the trivy scan for scan packages and remove snyk. The cron jobs and slack code updated as per last COA changes.
## Why?
To fix the node image vulnerabilities and update the drone pipeline with the latest changes
## How?
Updating .drone.yml, dockerfile and removing .snyk
## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

**Review by dk4g — APPROVED** [review]
All good. As discussed, can be merged once regression tests are passed.

File: .drone.yml [hunk 1]
```diff
@@ -423,68 +421,72 @@ steps:
...
+        SEVERITY: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL --dependency-tree
+        FAIL_ON_DETECTION: true
+        IGNORE_UNFIXED: false
+        ALLOW_CVE_LIST_FILE: hof-services-config/infrastructure/trivy/.trivyignore.yaml
     when:
       cron: security_scans
       event: cron
-
-  - name: cron_trivy_scan
+
```
Comment (Rhodine-orleans-lindsay) [inline .drone.yml:437]: could you remove the extra whitespace please?

File: Dockerfile [hunk 2]
```diff
@@ -20,7 +20,7 @@ RUN yarn install --frozen-lockfile --production --ignore-optional && \
     yarn run postinstall
 
 HEALTHCHECK --interval=5m --timeout=3s \
- CMD curl --fail http://localhost:8080 || exit 1
+    CMD curl --fail http://localhost:8080 || exit 1
```
Comment (Rhodine-orleans-lindsay) [inline Dockerfile:23]: could you remove the extra whitespace please?

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
Looks good 🙂 just a couple minor changes (see comments) to be addressed. It seems there is a small typo in your commits (HFF instead of HOFF). Could you also make sure when raising PRs that you have completed the checklist in the pr description?

File: .drone.yml [hunk 3]
```diff
@@ -423,68 +421,72 @@ steps:
     when:
       cron: security_scans
       event: cron
-
-  - name: cron_snyk_scan
-    pull: if-not-exists
-    image: node:18
+
```
[the remark this answers is not in the document]
Comment (gadorcruz) [reply .drone.yml:424]: It is a pipeline standard to keep spaces between steps, for example, in Drone CI documentation:
https://docs.drone.io/pipeline/overview/

File: Dockerfile (hunk 2 above)
Comment (gadorcruz) [reply Dockerfile]: Done

File: .drone.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (gadorcruz) [reply .drone.yml]: Done

**Pull request merged**